simpleclient_servlet/src/main/java/io/prometheus/client/exporter/MetricsServlet.java

```diff
@@ -14,6 +14,26 @@
 import java.util.HashSet;
 import java.util.Set;
 
+/**
+ * The MetricsServlet class exists to provide a simple way of exposing the metrics values.
+ *
+ * <p>This servlet might be used with a filter such the {@link io.prometheus.client.filter.MetricsFilter}
```

Comment:
That's for something else.

Reply:
This line shouldn't be here, exposition and instrumentation are separate concerns.